### PR TITLE
Add API Endpoint description for Get grouped trace metrics

### PR DESCRIPTION
**Pull request by joel-razor1**
Added a simple description.

### CODE REVIEW

**Comment by meghasalunke1** [comment]
@Koustav-Mukherjee1 Please help to review this PR. Thanks!

File: spec/descriptions/getTraceGroups.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Use this endpoint if one wants to get metrics for traces grouped under endpoint name or service name.
```
Comment (koustav-mukherjee) [inline spec/descriptions/getTraceGroups.md:1]:
```suggestion
The API endpoint retrieves metrics for traces that are grouped in the endpoint or service name.

```

File: spec/descriptions/getTraceGroups.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+<br/>
```
Comment (koustav-mukherjee) [inline spec/descriptions/getTraceGroups.md:2]:
```suggestion
```

File: spec/descriptions/getTraceGroups.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+**Note:** In this endpoint, the supported `groupbyTag` are `trace.endpoint.name` and `trace.service.name`.
```
Comment (koustav-mukherjee) [inline spec/descriptions/getTraceGroups.md:3]:
```suggestion
The supported `groupbyTag` are `trace.endpoint.name` and `trace.service.name`. 
{: note}

```

**Review by koustav-mukherjee — APPROVED** [review]

**Pull request merged**